interfaces/builtin/tmpfs_mount.go

```diff
@@ -0,0 +1,77 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2020 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+const tmpfsMountSummary = `allows mounting and unmounting TMPFS filesystems`
+
+const tmpfsMountBaseDeclarationSlots = `
+  tmpfs-mount:
+    allow-installation:
+      slot-snap-type:
+        - core
+    deny-auto-connection: true
+`
+
+const tmpfsMountConnectedPlugSecComp = `
+# Description: Allow mount and umount syscall access.
+mount
+umount
+umount2
```

Comment:
Over time, especially if the actual mount operation is done via libmount helpers, the mount call may be done with the new family of system calls. Hopefully those will be denied by seccomp and the denials will be handled by the library code but for an interface designed for mounting having a reminder TODO about the new mount API would be nice.

Reply:
:+1: